src/viam/services/vision/vision.py

```diff
@@ -1,29 +1,134 @@
 import abc
+import sys
 from typing import Any, Final, List, Mapping, Optional, Union
 
 from PIL import Image
 
-from viam.media.video import RawImage
+from viam.media.video import RawImage, ViamImage
 from viam.proto.common import PointCloudObject
-from viam.proto.service.vision import Classification, Detection
+from viam.proto.service.vision import Classification, Detection, GetPropertiesResponse
 from viam.resource.types import RESOURCE_NAMESPACE_RDK, RESOURCE_TYPE_SERVICE, Subtype
 
+if sys.version_info >= (3, 10):
+    from typing import TypeAlias
+else:
+    from typing_extensions import TypeAlias
+
 from ..service_base import ServiceBase
 
 
+class CaptureAllRequest:
+    """
+    CaptureAllRequest represents the collection of fields you can ask the
+    CaptureAllFromCamera method to return. If you request something that the service
+    cannot provide, a warning message will be generated, and the service will return an empty
+    version of the requested thing.
+    """
+    def __init__(self, return_image=False, return_classifications=False,
+                 return_detections=False, return_object_point_clouds=False):
+        """
+        Args:
+            return_image (bool): Return the image from the GetImage of the underlying camera.
+            Default is False.
+            return_classifications (bool): Return the classifications from GetClassifications.
+            Default is False.
+            return_detections (bool): Return the detections from GetDetections.
+            Default is False.
+            return_objet_point_clouds (bool): Return the point cloud objects from GetObjectPointClouds.
+            Default is False.
+
+        Returns:
+            None
+        """
+        self.return_image: bool = return_image
+        self.return_classifications: bool = return_classifications
+        self.return_detections: bool = return_detections
+        self.return_object_point_clouds: bool = return_object_point_clouds
```

Comment:
Mostly this is for future changes to the method. Given that it is Capture"All", what "All" may mean may change in the future. 

Let's say we add 2D segmentation as a vision service method. Then CaptureAll should be able to return it, too. But then it would be a breaking change to the method.

Actually, saying that outloud, python is nice in that you can assign defaults to arguments. So, if a future vision method was added, then the argument list could be extended then without breaking previous versions of the modules, because by default, the extra argument would be assigned a default value

Reply:
Essentially -- I was attempting to find a way to make it so there would not have to be a breaking change to the CaptureAll method if for some reason, a method was added to the vision service in the future